debian/control

```diff
@@ -12,3 +12,9 @@ Depends: ${shlibs:Depends}, ${misc:Depends}, python3, rpi-eeprom (>= 24)
 Description: Raspberry Pi USB booting code
  USB MSD boot code which should work on the Raspberry Pi model A,
  Compute Module, Compute Module 3, Compute Module 4, Compute Module 5, Raspberry Pi 5 and Raspberry Pi Zero.
+
+Package: make-boot-image
```

Comment:
Prefix with rpi-?

I'd like to avoid namespace clashes on the package name, if possible

Reply:
I wouldn't be against that - but it does mean it's a long type (or a quick tab-complete...)

Reply:
make-boot-image was never intended to become a generic tool  i.e. it was run within name space of a wrapper application so would not be in the global path.
If it's move to /usr/bin then it should have an rpi- prefix, otherwise if it's outside of a regular path then it doesn't matter what it's called

Reply:
FWIW, installing it under the name rpi-make-boot-image is exactly what we do in our Yocto recipe, exactly because make-boot-image is so generic, and we install it in the SDK that we use for all sorts of boards, not just the rpi family.